lib/plugins/aws/logs/README.md

```diff
@@ -0,0 +1,42 @@
+# Logs
+
+This plugin returns the CloudWatch logs of a lambda function. You can simply run `serverless logs -f hello` to test it out.
+
+## How it works
+
+`Logs` hooks into the [`logs:logs`](/lib/plugins/logs) lifecycle. It will fetch the CloudWatch log group of the provided function and outputs all the log stream events in the terminal.
+
+## Logs Options
+
+The logs plugin only require the function name you want to fetch the logs for. Other than that there are other options that you can add for extra control:
```

Comment:
I guess it's a current format but IMHO would be better to list all parameters and mark those which are required. It's easy to miss first line about function name.